labels/wallets.html

```diff
@@ -25,7 +25,7 @@ <h1>Wallets</h1>
              <th scope="col">NFT</th>
              <th scope="col">DID</th>
              <th scope="col">Offers</th>
-             <th scope="col">Memos</th>
+             <th scope="col"><abbr title="Air-gapped wallets are crypto wallets that are completely disconnected from the internet and any form of wireless communication. Providing additional security." class="featureQuestion">Air-gapped</abbr></th>
```

Comment:
I'd suggest rewording this to:

> Air-gapped wallets have the option to be completely disconnected from the internet and any form of wireless communication, providing additional security.

As to not suggest Pawket is *only* an airgapped wallet. Otherwise LGTM.